lib/ssh_pearl.sh

```diff
@@ -34,16 +34,19 @@ function ssh_pearl(){
 
     local rcScript=$(_aggregate_scripts "$PEARL_HOME/sshrc" "$PEARL_HOME/sshrc.d" | gzip | base64)
     local inputrcScript=$(_aggregate_scripts "$PEARL_HOME/sshinputrc" "$PEARL_HOME/sshinputrc.d" | gzip | base64)
+    local remoteTmpDirs=( "/tmp" "\$HOME" )
 
     CMD="
+        for tmpDir in ${remoteTmpDirs[*]}; do [[ -w \"\$tmpDir\" ]] && { foundTmpDir=\"\$tmpDir\"; break; } done
```

Comment:
Since the list of directory does not change dinamically. I would place the list explicitly here:

    for tmpDir in /usr \"\$HOME\"; do ..